src/tests/unit/tests/views/content/markup/code-example.test.tsx

```diff
@@ -65,4 +66,14 @@ describe('<CodeExample>', () => {
         const wrapper = shallow(<CodeExample>{`Line 1\nLine 2 [HIGHLIGHT\nHERE]Line 3\nLine 4`}</CodeExample>);
         expect(wrapper.getElement()).toMatchSnapshot();
     });
+
+    it('br has the correct key', () => {
```

Comment:
base on the title I would expect to see an assertion for the key, but found none. Should we be using snapshot at least?